docs/node/tutorial.mdx

```diff
@@ -0,0 +1,151 @@
+3. Add the second node to the network as a non-validating peer;
+4. Upgrade the second node to a validator.
+
+:::tip
+In order to do this tutorial, you will need to [install the `kwild` and `kwil-admin` binaries](<https://github.com/kwilteam/binary-releases/releases>).
+:::
+
+## Creating The First Node
+
+### Creating The Node's Files
+
+To create the node's configuration files, a new `genesis.json`, and a new keypair, we will use the `kwil-admin` tool:
+
+```bash
+kwil-admin setup init -o ./testnet/node0
+```
+
+### Running The Node
+
+Using the `kwild` binary, we can begun running the network with this single node:
+
+```bash
+kwild --root-dir ./testnet/node0 --app.admin-listen-addr unix:///tmp/node0.sock
+```
+
+The command then runs the local Kwil network, and begins producing blocks.
+
+### Getting The Node's Info
+
+In order to allow other nodes to join, we will need to expose our node's information publicly. Other operators can then set us as a peer,
+and request to join the network as a validator.
+
+To get our node's information, we can use the `kwil-admin` tool, specifying the UNIX socket that our admin RPC is listening on:
+
+```bash
+$ kwil-admin node status --rpcserver unix:///tmp/node0.sock
+{
+  "node": {
+    "chain_id": "kwil-chain-WfnGxpWd",
+    "name": "MacBook-Pro-7.local",
+    # highlight-next-line
+    "node_id": "b624b3395f22f8cb57cf86d58cbcd6b355d12e18", # node ID
+    "proto_ver": 8,
+    "app_ver": 0,
+    "block_ver": 11,
+    # highlight-next-line
+    "listen_addr": "tcp://127.0.0.1:26656", # p2p address
+    "rpc_addr": "tcp://127.0.0.1:26657"
+  },
+  "sync": {
+    "app_hash": "",
+    "best_block_hash": "",
+    "best_block_height": 0,
+    "best_block_time": "1969-12-31T18:00:00-06:00",
+    "syncing": false
+  },
+  "validator": {
+    "pubkey": "22cbbb666c26b2c1f42502df72c32de4d521138a1a2c96121d417a2f341a759c",
+    "power": 1
+  }
+}
+```
+
+The `node.node_id` and `node.listen_addr` fields can be given to the second node as a persistent peer, in the format: `node-id@p2p-address`.
+For the above node, we would have `b624b3395f22f8cb57cf86d58cbcd6b355d12e18@127.0.0.1:26656`.
+
+## Creating A Second Peer Node
+
+We will use the `kwil-admin` tool to create the second node's directories and configs. We can specify where it can find the genesis file,
+as well as peers that it should try to connect to when starting up. 
+
+```bash
+kwil-admin setup init --output-dir ./testnet/node1 --genesis ./testnet/node0/genesis.json \
+--peer b624b3395f22f8cb57cf86d58cbcd6b355d12e18@127.0.0.1:26656
+```
+
+To verify that the second node has been created correctly, we can start it up. Since we are running both of these locally, we need to specify
+new addresses for the different endpoints, so that the second node does not conflict with the first node:
+
+```bash
+kwild --root-dir=./testnet/node1 --app.grpc-listen-addr=localhost:50052 \
+--chain.p2p.persistent-peers=f29785e8f9c52635b55f6459b4b3a7330d3dee2e@127.0.0.1:26656 \
```

Comment:
Shouldnt this persistent peers address be the node id from the first node? the `b624b3395f22f8cb57cf86d58cbcd6b355d12e18@127.0.0.1:26656` peer address?